### ISSUE TITLE
Dark mode

### ISSUE THREAD

**Issue by HammyHavoc** [erxes/erxes#1942]
**Is your enhancement request related to a problem? Please describe.**
My colleague is legally blind and cannot use UX that features bright white. Dark mode is required for accessibility friendliness.

**Describe the solution you'd like**
Add a dark mode.

**Comment by Mambatukaa** [comment]
Hey there, 
Thank you for your suggestion. We've moved our enhancement/feature requests to https://roadmap.erxes.io
Please use this link in the future to submit your features. I've already copied and submitted this insight there.

Thank you.
